localenv/README.md

```diff
@@ -77,7 +73,7 @@ When clicking on the Account Name, a list of Transactions appears.
 
 - [Rafiki local environment setup](../README.md#environment-setup)
 - [docker](https://docs.docker.com/get-docker/)
-- [postman](https://www.postman.com/downloads/)
+- [Bruno](https://www.usebruno.com/downloads)
```

Comment:
```suggestion
- [Bruno](https://www.usebruno.com/downloads)
```
should we mention its an Open Source API client in here? to give some context without needing the reader to click in there?